WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt

```diff
@@ -240,21 +254,48 @@ class PageListViewModel @Inject constructor(
                     } else {
                         DEFAULT_INDENT
                     }
-                    PublishedPage(it.remoteId, it.title, it.date, labels, pageItemIndent,
-                            getFeaturedImageUrl(it.featuredImageId), actionsEnabled)
+                    val progressState = progressHelper.getProgressStateForPage(LocalId(it.pageId))
+
+                    PublishedPage(
```

Comment:
We might want to consider reverting these types of line-breaks we added in this PR (even the method declaration). Usually, I wouldn't mind too much, but since we are merging this into a feature branch and we plan to do a big clean-up before we merge the feature branch into develop. I think it'll make it easier for us to compare which components/parts make sense to re-use between PostList and PageList. Wdyt?

Reply:
Thanks. Is there any way I can modify Android Studio to avoid this because auto formatting introduces it at times. I will double-check diff for them.

Reply:
- Preferences -> Editor -> Code Style -> HardWrap at
- Preferences -> Editor -> Code Style -> Kotlin -> Wrapping and Braces -> HardWrap at

Reply:
Ohh I posted the reply to a wrong comment :D.

I usually just select the method I added and run the auto-formatter only on the method.

Reply:
Okay, thanks!

Reply:
I will do a final run-through tomorrow again to resolve all of these. When I added the extra parameters and did the formatting they were introduced even though I ran it on the line only. I think it's because they can't hold in a single line it opted to put each on its own line.